packages/common/client-utils/package.json

```diff
@@ -59,8 +61,10 @@
 		"test": "npm run test:mocha && npm run test:jest",
 		"test:coverage": "c8 npm test",
 		"test:jest": "jest",
-		"test:mocha": "mocha  --recursive \"dist/test/mocha/**/*.spec.*js\" --exit",
-		"tsc": "tsc",
+		"test:mocha": "npm run test:mocha:esm && npm run test:mocha:cjs",
```

Comment:
nit: do we want to skip cjs test?

Reply:
Not for this one. This packages covers complicated things and we should test CJS for it at least for now.